### DISCUSSION TITLE
feat: Time of create filtering.

### DISCUSSION THREAD

**Discussion by furrki** [acikkaynak/deprem-yardim-frontend#552] [Ideas]
## Task Definition 
Having a date picker on top right corner, and filtering by date feature

## Other Evaluated Alternatives
Buttons like last day, last 24 hours can be added.

## Final Notes
First 3 days of the disaster is critical, later on survival possibility is low. In the meantime if can provide this filtering feature, it will be helpful to find victims.

**Comment by cdagli** [comment]
It's implemented by backend team, so I think we can swiftly do this.

https://github.com/acikkaynak/deprem-yardim-backend/pull/90

Initial request can be 24 hours, then dropdown can have an option for "all" to keep it simple.

**Comment by teomanofficial** [comment]
Later on we can have other filtering options, we need a different solution in terms of UX.

**Comment by khnbzkrt** [comment]
@teomanofficial hocam size assign edilmiş birçok iş görünüyor isterseniz ben de üzerime alabilirim bu geliştirmeyi.

**Comment by teomanofficial** [comment]
@khnbzkrt We should handle this differently, Whe should implement a filter icon on the corner and then we can add different search parameters. I'll be creating a new discussion, let move it over there.

Discussion: https://github.com/acikkaynak/deprem-yardim-frontend/discussions/485

**Comment by cdagli** [comment]
Related: https://github.com/acikkaynak/deprem-yardim-backend/issues/138

**Comment by 0xCakin** [comment]
Good idea, and this has been requested by NGOs as well. Let's forward it to UX/UI.

**Comment by cdagli** [comment]
Backend team is on it. Let's convert it to issue and it would be great if we can start asap.

**Reply by 0xCakin** [reply]
@cdagli start working on it, we should distribute it and then let's create an issue.

**Comment by berkaydesign** [comment]
Design team is Ok'd @canerakin111 , @furrki , @oguzergul 
If there is another request made lets do that as well. This was the very first issue requested from design team.
If you like to talk over it, find me on discord wiser#1809 or "tasarım / UX & UI" chat room. 

Here is the figma link: https://www.figma.com/file/sctw6xtcdoFOfmE0gC97Ft/Afetharita.com?node-id=2349%3A8093&t=vQfzb1258VlyQMC6-1
**It is read as art board in  "HAZIR (FRONTEND BURAYA)" section, on first page.**

**Comment by 0xCakin** [comment]
released.
